lightly/data/collate.py

```diff
@@ -1176,15 +1193,17 @@ class VICRegLCollateFunction(nn.Module):
         local_gaussian_blur_prob:
             Probability of Gaussian blur for the local crop category.
         global_gaussian_blur_kernel_size:
-            Old argument. Value is deprecated in favor of sigmas.
+            Will be deprecated in favor of `sigmas` argument. If set, the old behavior applies and `sigmas` is ignored.
```

Comment:
sigmas -> global_gaussian_blur_sigmas

Please also use the correct variable names when explaining how the sigma is calculated. Same for the docstrings below (local_gaussian_blur_kernel_size, global_gaussian_blur_sigmas, local_gaussian_blur_sigmas)